e2e/volume_ops_test.go

```diff
@@ -271,6 +272,39 @@ func testVolumeDelete(t *testing.T) {
 func testVolumeStart(t *testing.T) {
 	r := require.New(t)
 	r.Nil(client.VolumeStart(volname, false), "volume start failed")
+
+	vol, err := client.BricksStatus(volname)
+	for index := range vol {
+		r.NotZero(vol[index].Port)
+		r.True(vol[index].Online)
+	}
+
+	vol, err = client.BricksStatus(volname)
```

Comment:
The returned type if not `Volinfo` and hence this variable can be better named as `bricks` to indicate the type `api.BricksStatusResp,`